examples/hand_pose_estimation/backend_keypoints.py

```diff
@@ -0,0 +1,1163 @@
+
+    # Arguments
+        uv_coordinates: Numpy array of shape (21, 2).
+        scoremap_size: List of size (2).
+
+    # Returns
+        X_limits: Numpy array of shape (21, 1).
+        Y_limits: Numpy array of shape (21, 1).
+    """
+    shape = uv_coordinates.shape
+
+    x_range = np.expand_dims(np.arange(scoremap_size[0]), 1)
+    x_coordinates = np.tile(x_range, [1, scoremap_size[1]])
+    x_coordinates.reshape((scoremap_size[0], scoremap_size[1]))
+    x_coordinates = np.expand_dims(x_coordinates, -1)
+    x_coordinates = np.tile(x_coordinates, [1, 1, shape[0]])
+    x_limits = x_coordinates - uv_coordinates[:, 0].astype('float64')
+
+    y_range = np.expand_dims(np.arange(scoremap_size[1]), 0)
+    y_coordinates = np.tile(y_range, [scoremap_size[0], 1])
+    y_coordinates.reshape((scoremap_size[0], scoremap_size[1]))
+    y_coordinates = np.expand_dims(y_coordinates, -1)
+    y_coordinates = np.tile(y_coordinates, [1, 1, shape[0]])
+    y_limits = y_coordinates - uv_coordinates[:, 1].astype('float64')
+
+    return x_limits, y_limits
+
+
+def create_multiple_gaussian_map(uv_coordinates, scoremap_size, sigma,
+                                 validity_mask):
+    """ Generate Gaussian maps based on keypoints in Image coordinates.
+
+    # Arguments
+        uv_coordinates: Numpy array of shape (21, 2).
+        scoremap_size: List of size (2).
+        sigma: Integer value.
+        validity_mask: Integer value.
+
+    # Returns
+        scoremap: Numpy array of shape (256, 256).
+    """
+    assert len(scoremap_size) == 2
+    keypoints_mask = get_keypoints_mask(
+        validity_mask, uv_coordinates, scoremap_size)
+    x_limits, y_limits = get_keypoint_limits(uv_coordinates, scoremap_size)
+    squared_distance = np.square(x_limits) + np.square(y_limits)
+    scoremap = np.exp(-squared_distance / np.square(sigma)) * keypoints_mask
+    return scoremap
+
+
+def keypoint_xy_coordinates(shape):
+    """ Generate X and Y nesh.
+
+    # Arguments
+        shape: tuple of size (3).
+
+    # Returns
+        X: Numpy array of shape (1, 256).
+        Y: Numpy array of shape (256, 1).
+    """
+    x_range = np.expand_dims(np.arange(shape[1]), 1)
+    y_range = np.expand_dims(np.arange(shape[2]), 0)
+    x_coordinates = np.tile(x_range, [1, shape[2]])
+    y_coordinates = np.tile(y_range, [shape[1], 1])
+    return x_coordinates, y_coordinates
+
+
+def get_bounding_box_list(X_masked, Y_masked):
+    """ Get Bounding Box.
+
+    # Arguments
+        X_masked: tuple of size (256, 1).
+        Y_masked: tuple of size (256, 1).
+
+    # Returns
+        bounding_box: List of length (4).
+    """
+    x_min, x_max = np.min(X_masked), np.max(X_masked)
+    y_min, y_max = np.min(Y_masked), np.max(Y_masked)
+    start = np.stack([x_min, y_min])
+    end = np.stack([x_max, y_max])
+    bounding_box = np.stack([start, end], 1)
+    return bounding_box
+
+
+def get_center_list(box_coordinates, image_center=[160, 160]):
```

Comment:
Is not a list. Also why are you passing those default values? Maybe remove those default values.